labscommunity/arweavekit
# feat: add .nvmrc to unify development envs; add main ci to run tests

## Summary

- Added `.nvmrc` file to unify collab efforts (currently set to `18` so we use Node 18.x)
- Added skim contributing guide to bottom of README
- Added workflow to run tests on `main` (uses Node 16, 18, 20)

## How to test

- Install `nvm` at https://github.com/nvm-sh/nvm
- Pull down this branch
- In the project's root directory, run `nvm use`. Your terminal should say it's using Node 18.x (the version in the `.nvmrc` file).
